lab-katherine/__test__/auth-router.test.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+require('./lib/setup.js');
+const superagent = require('superagent');
+const server = require('../lib/server.js');
+const accountMock = require('./lib/account-mock.js');
+
+const apiURL = `http://localhost:${process.env.PORT}`;
+
+describe('AUTH router', () => {
```

Comment:
This is a great set of test! awesome work!. I would just add a little more detail in the describe messages (i.e. when will a test pass or fail) so that's easier to read by other members of a team.